### PR TITLE
Print helm install command after publishing nightly chart

### PR DESCRIPTION
**Pull request by alexander-demicev**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**:

- [ ] squashed commits into logical changes
- [ ] includes documentation
- [ ] adds unit tests
- [ ] adds or updates e2e tests

### CODE REVIEW

File: .github/workflows/nightly-chart-and-image-publish.yaml [hunk 1]
```diff
@@ -111,10 +115,10 @@ jobs:
         with:
           version: 3.8.0
       - name: Build Helm chart
-        run: make release-chart RELEASE_TAG=${{ env.TAG }}
+        run: make release-chart RELEASE_TAG=${{ env.TAG }} CONTROLLER_IMAGE_VERSION=${{ env.TAG }}
```
Comment (Danil-Grigorev) [inline .github/workflows/nightly-chart-and-image-publish.yaml:118]: Does this controller image version do anything here?

File: .github/workflows/nightly-chart-and-image-publish.yaml [hunk 2]
```diff
@@ -111,10 +115,10 @@ jobs:
...
-        run: make release-chart RELEASE_TAG=${{ env.TAG }}
+        run: make release-chart RELEASE_TAG=${{ env.TAG }} CONTROLLER_IMAGE_VERSION=${{ env.TAG }} 
       - name: Login to prod registry using Helm
         run: |
           echo ${{ secrets.REGISTRY_PASSWORD }} | helm registry login ${{ secrets.REGISTRY_ENDPOINT }} --username ${{ secrets.REGISTRY_USERNAME }} --password-stdin
       - name: Publish Helm chart as OCI image to prod registry
         run: |
-          helm push out/package/rancher-turtles-0.0.0-${{ github.sha }}.tgz oci://${{ secrets.REGISTRY_ENDPOINT }}/${{ env.PROD_ORG }}/rancher-turtles-chart
\ No newline at end of file
+          helm push out/package/rancher-turtles-0.0.0-${{ github.sha }}.tgz oci://${{ secrets.REGISTRY_ENDPOINT }}/${{ env.PROD_ORG }}/rancher-turtles-chart
```
Comment (Danil-Grigorev) [inline .github/workflows/nightly-chart-and-image-publish.yaml:124]:
```suggestion
          helm push out/package/rancher-turtles-${{ env.TAG }} tgz oci://${{ secrets.REGISTRY_ENDPOINT }}/${{ env.PROD_ORG }}/rancher-turtles-chart
```

File: .github/workflows/nightly-chart-and-image-publish.yaml (hunk 1 above)
Comment (alexander-demicev) [reply .github/workflows/nightly-chart-and-image-publish.yaml:118]: Yes, we want to set imageVersion in values.yaml

Comment (Danil-Grigorev) [reply .github/workflows/nightly-chart-and-image-publish.yaml:118]: The tag is used for that, this variable is not present in the code

Comment (alexander-demicev) [reply .github/workflows/nightly-chart-and-image-publish.yaml:118]: https://github.com/rancher-sandbox/rancher-turtles/blob/main/Makefile#L480, currently this value is empty and chart doesn't install

**Review by Danil-Grigorev — APPROVED** [review]

**Review by richardcase — APPROVED** [review]

**Pull request merged**